### DISCUSSION TITLE
Update tags of container images

### DISCUSSION THREAD

**Discussion by anoordover** [CommunityToolkit/Aspire#317] [General]
Should we update tags of the container images being used to the latest version?
e.g. azure-databases/data-api-builder is now on version 1.3.19.
ollama/ollama version 0.5.1
open-webui/open-webui v0.4.8

**Comment by aaronpowell** [comment]
Yep, they need to be bumped for the next release

**Reply by anoordover** [reply]
What do you want:
- Create a PR for every bump?
- Create an issue for every bump?

**Reply by aaronpowell** [reply]
Individual PRs are good because it means we don't get overloaded with too many things to verify in each review.
